### PR TITLE
Bugfix and minor adjustments

### PR DESCRIPTION
**Pull request by Desolane**
commit history goes over changes made

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -493,87 +493,102 @@
...
+/obj/structure/closet/crate/wwi
 	icon = 'icons/FoF/misc.dmi'
-	can_hold = list(/obj/item/weapon/gun/projectile,/obj/item/ammo_magazine,/obj/item/clothing,/obj/item/weapon/melee/combat_knife,/obj/item/clothing/mask/gas,/obj/item/ammo_box,/obj/item/clothing/head/helmet)
+	icon_state = "wcrate"
+	icon_opened = "wcrate_o"
 	w_class = ITEM_SIZE_HUGE
 
-/obj/item/weapon/storage/box/wwi/germ
-	icon_state = "flatcrate"
+/obj/structure/closet/crate/wwi/germ
```
Comment (Quantomicus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Is there a reason for this empty definition to exist?

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -493,87 +493,102 @@
...
 
-/obj/item/weapon/storage/box/wwi/brit
-	icon_state = "lightflatcrate"
+/obj/structure/closet/crate/wwi/brit
```
Comment (Quantomicus) [inline code/game/objects/items/weapons/storage/boxes.dm]: Same question.

File: icons/FoF/code/items_objects.dm [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/obj/item/weapon/reagent_containers/pill/iron
+	name = "Iron pill"
+	desc = "Used to increase the speed of blood replenishment."
+	icon_state = "pill18"
+	New()
```
Comment (MadmanMartian) [inline icons/FoF/code/items_objects.dm]: partial pathing is bad

Comment (Desolane) [reply icons/FoF/code/items_objects.dm]: literally chopped out of a dm specifically to our upstream. will fix.

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (Desolane) [reply code/game/objects/items/weapons/storage/boxes.dm]: to classify subtypes under a dropdown tree in the object trees. it serves to purpose other than consolidation for mapping. can remove if that's what's wanted.

Comment (Quantomicus) [reply code/game/objects/items/weapons/storage/boxes.dm]: That's fine. Never seen that before, but it makes a lot of sense.

File: icons/FoF/code/items_objects.dm (hunk 3 above)
Comment (Quantomicus) [reply icons/FoF/code/items_objects.dm]: Missed that one, thanks.

**Pull request merged**